Roles/Neutral/God.cs

```diff
@@ -46,7 +46,7 @@ public override bool GuessCheck(bool isUI, PlayerControl guesser, PlayerControl
         return false;
     }
 
-    public override void OnMeetingHudStart(PlayerControl pc)
+    public override void OnOthersMeetingHudStart(PlayerControl pc)
```

Comment:
Same here, the code is designed only for God and the message is sent to everyone if necessary